src/test/ui/consts/min_const_fn/min_const_fn.rs

```diff
@@ -108,17 +108,14 @@ const fn foo37(a: bool, b: bool) -> bool { a || b }
 fn main() {}
 
 impl<T: std::fmt::Debug> Foo<T> {
-//~^ ERROR trait bounds other than `Sized` on const fn parameters are unstable
```

Comment:
This part of the diff seems wrong to me (this is also what happened in `proc_macro`).

Reply:
I think the is may need its own test. The message is missing because other errors are reported first?

Reply:
That makes no sense to me, the order of errors shouldn't change just because I got rid of a feature flag...